README.md

```diff
@@ -280,6 +280,14 @@ Some [Serverspec](https://serverspec.org/) extensions are shipped and enabled by
 Often you want to test some service that exposes things over HTTP.
 Instead of using [`command("curl …")`](https://serverspec.org/resource_types.html#command) you can use `curl_command(…)` which behaves like a Serverspec `command` but adds matchers for the HTTP response code and the response body.
 
+4 different parameters are supported right now:
+
+* headers ( hash of header options
+* user (accepts username:password for basic auth)
```

Comment:
Do we need to specify this? Would it be better to make it clear users should read the man page about `--user`?

Reply:
I updated it to point to the manpage